packages/datadog-plugin-aws-sdk/test/sqs.spec.js

```diff
@@ -268,6 +296,176 @@ describe('Plugin', () => {
           }, 250)
         })
       })
+
+      describe('data stream monitoring', () => {
+        const expectedProducerHash = computePathwayHash(
```

Comment:
This should be run manually and the output should be used directly in the test. This is because otherwise if there is a bug in `computePathwayHash`, then the test will still pass since it will be validating that the bugged result is used.

Reply:
Similar idea for `ENTRY_PARENT_HASH`.

Reply:
Basically, the implementation should not be used to test the implementation.

Reply:
This test is used to ensure that the pathway hash is injected to the span's tags. Testing for `computePathwayHash` is already included in the explicit testing for datastreams [here](https://github.com/DataDog/dd-trace-js/blob/81e3d687db625d8aa1ba5c5bb17d75ab9bde4f9e/packages/dd-trace/test/datastreams/pathway.spec.js#L14), which will catch any changes to the logic computing hash as the output hash is explicit.

Reply:
I usually prefer tests with a deterministic output that doesn't depend on the code itself, but if you're confident in the other tests then I'm not going to block on this.

I will mention however that this is how pprof-format broke at some point, its unit tests were passing, but the actual end result had a bug that resulted in the encoding being completely broken in certain unforeseen scenarios.

Reply:
Since i have no strong opinions on this, I've changed all my new tests to instead use the `computePathwayHash` output as you earlier suggested.